scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixOutput.java

```diff
@@ -0,0 +1,66 @@
+package scalafix.interfaces;
+
+import java.nio.file.Path;
+import java.util.Optional;
+
+public interface ScalafixOutput {
```

Comment:
Same comment as before, I find it weird to have an `Output` class (a very passive name) from where we can trigger side effects. I am not sure my suggestion is better though
```suggestion
public interface ScalafixFileEvaluation {
```

Reply:
At least evaluation is consistent with the naming we decide for ScalafixEvaluation. So I prefer your naming